ArtOfIllusion/src/artofillusion/ArtOfIllusion.java

```diff
@@ -136,7 +120,7 @@ public static void main(String args[])
     {
       UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
     }
-    catch (Exception ex)
+    catch (ClassNotFoundException | IllegalAccessException | InstantiationException | UnsupportedLookAndFeelException ex)
```

Comment:
Why limit to certain classes of exceptions? Do you expect others to bubble up and be more appropriately caught elsewhere?